nucleus/payara-modules/healthcheck-core/src/main/java/fish/payara/nucleus/healthcheck/preliminary/LocalStrings.properties

```diff
@@ -43,4 +43,5 @@ healthcheck.description.garbageCollector=Provides ratio on garbage collection co
 healthcheck.description.heapMemory=Provides ratio on used heap memory with severity according to defined threshold values
 healthcheck.description.hoggingThreads=Lists hogging threads with their id when given thresholds exceed
 healthcheck.description.machineMemory=Provides ratio on used machine memory with severity according to defined threshold values
-healthcheck.description.stuckThreads=Provides thread name, id and stack trace for requests which reach over defined threshold values
\ No newline at end of file
+healthcheck.description.stuckThreads=Provides thread name, id and stack trace for requests which reach over defined threshold values
+healthcheck.description.MPhealthcheck=Checks that all instances are responding to Microprofile Healthcheck requests with an UP response
```

Comment:
Don't think block capitalisation of _up_ is necessary here

Reply:
The MP Health spec defines it as UP with capitalisation.